lib/hyper/core/service/configuration.rb

```diff
@@ -7,10 +7,15 @@ class Configuration
         API_VERSION = 3
         PRODUCT = 'engage'
         NON_VERSIONED_API_PATH = 'visualize'
+        PROTOCOL = 'https://'
```

Comment:
How about using DEFAULT_PROTOCOL as name here? Also, should the protocol include "://" or just the "https".

Reply:
You are right, i will do it.